src/components/pegout/AddressDialog.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <v-dialog v-model="showAddressDialog" width="520" persistent>
+    <v-card class="container dialog">
+      <v-col cols="12" align-self="center" class="pt-0">
+        <v-row class="mx-0 mt-3 mb-3 d-flex justify-center">
+            <p class="pop-up-title">Sign this message</p>
+        </v-row>
+        <v-row class="mx-0 mb-10 justify-center">
+            <p class="pop-up-subtitle mb-2">
+              We will the signature to device your Bitcoin address.
+            </p>
```

Comment:
Fix the typo : `We will use the signature to derive your Bitcoin address.`

Reply:
solved.